Main.java

```diff
@@ -9,7 +11,25 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // Creat a Scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    // Asks the user to input a number
+    System.out.println("Please enter an integer:");
+
+    // gets the number from the user
+    int number = input.nextInt();
+
+    // identifies if a number is odd
+    final int odd = 1;
+
+    // identifies if number is even or odd
+    if (number % 2 == odd){
+    // tells the user the number is odd
+    System.out.println(number + " is an odd number");
```

Comment:
Good work but watch your indentation. If you have a statement inside of an if-statement block, it should be another TAB over,

```
if (number % 2 == odd){
      // tells the user the number is odd
      System.out.println(number + " is an odd number");
}
```